dev/service-registry/service-locator/src/main/scala/com/lightbend/lagom/gateway/NettyServiceGateway.scala

```diff
@@ -117,14 +113,6 @@ class NettyServiceGateway(coordinatedShutdown: CoordinatedShutdown, config: Serv
     }
   }
 
-  private implicit def ecFromNettyExecutor(executor: EventExecutor): ExecutionContext = new ExecutionContext {
```

Comment:
Was this implicit supposed to be removed?

Reply:
Nice catch. 

Mental note: I don't know if we still have a `scripted` test that used `netty` but I don't think we do. If there's no `scripted` test for Netty GW I wouldn't create it, just curious.